libraries/abstractions/common_io/include/iot_i2c.h

```diff
@@ -178,29 +179,30 @@ void iot_i2c_set_callback( IotI2CHandle_t const pxI2CPeripheral,
  * @param[in] xBytes The number of bytes to read.
  *
  * @return
- * - IOT_I2C_SUCCESS, on success and all the requested bytes have been read
```

Comment:
This is what I meant to comment on, is it necessary to remove this line?

Reply:
Yeah, I removed because I think partial read could happen.

Reply:
hmm I assumed it would return Timeout if all the request bytes weren't sent

Reply:
Oh, you are right. Let me add those comments back.